src/packages/core/protocol/__tests__/account.spec.ts

```diff
@@ -13,6 +13,13 @@ describe('create new accounts', () => {
  
 	describe('valid private key', () => {
 
+		// describe('balance', () => {
```

Comment:
Please, add some unit tests for this new feature of the Account class

Reply:
Added tests